src/common-utils/main.sh

```diff
@@ -408,10 +408,15 @@ fi
 if [ "${USERNAME}" = "root" ]; then
     user_home="/root"
```

Comment:
Can we bump up the minor version?

https://github.com/devcontainers/features/blob/ed0060ebc48e948f8ea11270a0c06fa840926a1b/src/common-utils/devcontainer-feature.json#L3

Reply:
bumped to version 2.2.0